src/app/components/PageLayout/Logotype.tsx

```diff
@@ -9,7 +9,7 @@ import { useTranslation } from 'react-i18next'
 
 interface LogotypeProps {
   color?: string
-  showText: boolean
+  showText?: true | false | undefined // If undefined: shows text on desktop
```

Comment:
Do we need both the question mark _and_ the undefined value? Seems a bit redundant...

Reply:
Nope